### ISSUE TITLE
Is Wakanda dying?

### ISSUE THREAD

**Issue by chriscurnow** [Wakanda/wakanda-issues#95]
I am starting to get very concerned about the future of Wakanda.

There has only been one issue closed here since July 18.

We have had v1.1.0 in beta since July and I haven't seen any updates.

Overall there just seems to be a significant decline in the activity around Wakanda.

I hope I'm wrong. Please let me be wrong. I really like Wakanda.

**Comment by vegarringdal** [comment]
Think they lost a lot when they closed the wakanda forum.
Also the the server had it issues memory leak issues, random quits.
I tried once disabling the http server and just using the rest api, didnt see the huge spikes in memory when doing that, but it was some very simple tests

I really liked the entity-set part/ with collection methods. Wonder if I can create something like that with nodejs/postgres, wont be able to create what I wanted to create without that, not in a good way at least.

Maybe they should just focus on the database/rest API, model creator was nice, maybe that could have been a part of the server admin, bringing all manpower to 1 thing, super simple database thats very easy to use/and get started with. 

New name and new blank forum would be nice too.

But I havent used it for a long time, so maybe some parts is better, I really do not know.

One thing I missed was a better way of doing reports, a SQL option into the server could have helped, then you would not have to loop huge collections or create  unwanted classes.

**Comment by hamzahik** [comment]
TL;DR: Wakanda is doing fine :)

Hello @chriscurnow, appreciate your feedback as always !

Let me give you an update about Wakanda ( I admit we've been quiet for some time, quiet but not dead, working on lots of cool stuff :) )

The 1.1.0 will be released this week (tomorrow for the community if all the installers' checks are green and the enterprise version a day later)

We will be present at ngEurope (Oct 2016) in Paris as we were in ngConf in the US (May 2016), we are even the official sponsor of the [Hackaton ](https://twitter.com/ngEurope/status/776733108762738689) and @intissarMez will give a talk about Wakanda on a local Angular event in Rabat Morocco late September.

Following Wakanda on Twitter is a good way to keep in touch : events, releases, blog posts.. are all announced there and to help other wakanda developers or to get some help you can use [StackOverflow](http://stackoverflow.com/search?tab=newest&q=wakanda).

> There has only been one issue closed here since July 18.

We have [**11**](https://github.com/Wakanda/wakanda-issues/issues?q=is%3Aopen+is%3Aissue+label%3Abug) issues marked as bugs most of them are fixed in the 1.1.0 (to be released as I said this week). Keeping the issues open until an official release (and not a pre-release) is available was a community request so we will keep those issues open until the 1.1.0 is available for download.

We are working on big changes in Wakanda both on the product and communication levels : 

**Communication**
- Improved and unified download process to address #89 and other issues ( @cedricss  is helping us on this one )
- A better backend [documentation ](http://wakanda.github.io/wakanda-server-doc/#/doc/overview) (Work in Progress, but please do give us your feedback)
- Publicly available roadmap

**Product**

**1.1.0**

This release will contain : 
- `directory.setSessionManager` to help scale your Wakanda apps -Enterprise Version Only- ([more ](http://wakanda.github.io/wakanda-server-doc/#/doc/authentication/custom-session-management)on this)
- `httpServer.request` and `httpServer.response` to access request and response objects from anywhere even from a login listener or DataStore methods when called via REST (discussion on [SO
](http://stackoverflow.com/questions/36284927/expose-application-to-ip-range-or-validate-a-login-requet-against-an-ip-range/36297164#comment60237902_36290099  @steeu )
- New admin interface - use http://127.0.0.1:8080/admin/ to access it- (We will keep both the old and the new interface until the new one covers the necessary features)
- WebSockets high CPU & memory consumption Fix
+ Lots of fixes (details to be published in the updated changelog)

As soon as the 1.1.0 is released we will make the major release **2.0.0** available for public testing.

**2.0.0**

- Multi-Model : Using a wizard you can add remote/local data sources ( Wakanda, 4D, MySQL, ODBC, Virtual ) every model is accessible via a separate model file and its own events..
- NodeWorker : The same as  a SharedWorker but you can use **NodeJS** APIs inside of it ! The first pre-release will contain the equivalent of ioJS but the official 2.0.0 should expose a NodeWorker equivalent to **NodeJS V6 LTS**
- Studio WebViews will use Blink instead of WebKit : this will solve lots of problems ! (Live reload for winows, the debugger can be opened inside the studio.. )
- Login Manager : an alternative for the login listener API to avoid creating the `required.js` file
- Admin Interface Job Runner can now show execution logs, and is equiped with a calendar view
..

Hope this gives you more insight on the project.

**Comment by vegarringdal** [comment]
@hamzahik 
Nice post, kinda want to give it a try again, some questions if you have time.

**Rest API**
Is there any upcoming changes to the REST API?
Will anything be removed, or added, or alternative REST api ?
Kinda want to make sure I dont download and make soemthing and one of the things I need like entityset & selection gets killed :-)


**WebSockets high CPU & memory consumption Fix**
I remember I had few issues what that, esp IE was making CPU go crazy


**NodeWorker:**
Sounds very cool.
Will the node be built in, or can you select what version to use?


**Lisence:**
Whats can you do with the free version vs enterprise?
Is there a list showing the difference?


**Gitter**
Why not have something like gitter channel to gather the community more, so its easier for users to connect, thats one of the things you lost with the forum, the forum had a lot of garbage so I understand the need to get rid of it too.
Gitter have been very helpful while learning Aurelia and getting connected with other users


**Docs:**
Whats the official docs, is it still doc.wakanda.org?

**Comment by northernfrontier** [comment]
Sounds great looking forward to all the new features, never had any doubts you were all busy making things better.


gary


------------------------------------------------------------------------
Gary Pedretty                                        gary@ravnalaska.net <mailto:gary@eraalaska.net>
Systems Manager                                          www.flyravn.com <http://www.flyravn.com/>
Ravn Alaska                           /\                    907-450-7251
5245 Airport Industrial Road         /  \/\             907-450-7238 fax
Fairbanks, Alaska  99709        /\  /    \ \ Second greatest commandment
Serving All of Alaska          /  \/  /\  \ \/\   “Love your neighbor as
Really loving the record green up date! Summmer!!   yourself” Matt 22:39
------------------------------------------------------------------------

**Comment by alcada** [comment]
Thanks @hamzahik for the detailed information on wakanda coming features.
Now I'm much more excited about the future of the platform.
I only have two main concerns:
1-Does the team plans to give a similar update/improve with respect to the web page design?
2-For when new updated V2 widgets fully functional (like the old ones in wakanda 10).
Great Job
Luís Alçada
Coimbra University

**Comment by chriscurnow** [comment]
@hamzahik – that's pretty much what I was hoping to hear.
I really like the Wakanda server and would hate to see anything that threatened its continued existence.
It all sounds very exciting and I'm very much looking forward to it all.
Thanks so much for the detailed info.
Let us know if there is anything we can do to support the platform.
Best regards
Chris

**Comment by AaronLavers** [comment]
Can also confirm, there's wind in the sails and the ships going in the right direction.

**Comment by bea2701** [comment]
Wakanda will be back on the market perhaps even bigger then before, i am sure

**Comment by hamzahik** [comment]
The 1.1.0 community release is available now for [download ](https://github.com/Wakanda/wakanda-digital-app-factory/releases/tag/v1.1.0 ) on Github it will be official once the documentation is ready.
The enterprise version should be available a day later.

**Comment by chriscurnow** [comment]
@hamzahik – that's great news. I'm downloading it now:)

**Comment by chriscurnow** [comment]
Hmm, @hamzahik,  I'm still not seeing a great deal of activity around Wakanda.

The new improved documentation you refer to above looks good as a format but hasn't been updated for 2 months.

We've moved from 1.1.0 to 1.20 with only a handful of fixes between the two.

You announce news and updates on Twitter – but I don't use Twitter that much. I find it a distraction.

Why can't your Twitter feed be included on the website so we can go an find it really quickly and easily.

Overall, it doesn't feel like your communication has improved from when I started this thread.

Best regards
Chris

**Comment by hamzahik** [comment]
Hello @chriscurnow,

Since last time we spoke about the subject, the social accounts of Wakanda (including twitter) became very active and we started working on improving our [blog ](http://blog.wakanda.io), the blog is being  improved and regular technical posts about Wakanda and web & Mobile dev in general are scheduled (for very soon).

To be frank, all the communication efforts we are making are targeted for the 2.0 version since we think that'll be the future of Wakanda, if we communicate too much on the current version (which already contains lots of improvements compared to older ones) this might cause frustration for new comers (lack of documentation, no NodeJS support..)

Your idea of integrating the twitter feed on the home page is great, we were going to integrate the [gitter ](https://gitter.im/wakandasoft/Lobby) channel so why not do the same for twitter :)

Not counting the pre-releases, we have shipped 6 versions of Wakanda this year, compared to the last year, that is an important step forward.

As promised, the 1.1.0 was released with all the features that we've announced before hand. For the 2.0 we can give the community a pre-release but the "new webview" part is causing a great deal of instability so we decided to delay a bit the announcement of a pre-release.

The documentation is I said a work in progress, not everything was published yet, and we didn't release new features since the 1.1.0 so there was no need to document something new. We are preparing the documentation for the 2.0 and we hope everyone will be satisfied with the content.

I know that from the outside things might seem slow, but I assure you that a lot is being done and prepared so the 2.0 can be a release everyone is happy with.

Thanks a lot for your feedback, it means a great deal to the team.

Best regards,

Hamza

**Comment by chriscurnow** [comment]
Hi @hamzahik,

That's great to hear Hamza.

I really appreciate you taking the time to respond so quickly. And I am really looking forward to seeing 2.0. I'll read more about it when I have a chance. (I'll be taking some time off straight after Christmas so I'll be able to catch up with what's available then.)

Best regards,
Chris

**Comment by chriscurnow** [comment]
Hi again @hamzahik ,

I hadn't caught up with your Gitter room. That's fantastic.

Have you thought about using Slack as well. A lot of tech teams seem to use it as well as Gitter.

Best regards
Chris

**Comment by bea2701** [comment]
Has  a time table been set /available for V2.0 release eq. q1-q2 2017, how can (developers) make this a to become a final succes we help, beta testing or otherwise ?

**Comment by hamzahik** [comment]
@bea2701 Yes we will very soon put a non-official version online for people who want to try out some of the new features. But it won't be the final version because lots of things are still in preparation.

The official release of 2.0 is scheduled for late Q1

**Comment by hamzahik** [comment]
@chriscurnow for now we are sticking with gitter. Depending on the need we might need to use another tool.
